assets/app/view/spreadsheet.rb

```diff
@@ -96,14 +144,62 @@ def render_title
       ]
     end
 
+    def sort_order_icon
+      return '(⬇️)' if @spreadsheet_sort_order == 'ASC'
+
+      '(⬆️)'
+    end
+
+    def mark_sort_column(sort_by)
+      Lib::Storage['spreadsheet_sort_by'] = sort_by
```

Comment:
i don't think you need any needs if you're using local stoarge

so just do

Lib::Storage['spreadsheet_sort_by'] = sort_by
update

do you want this to be user global or per game? i think user global makes ssense

Reply:
I think global also makes sense. So yeah `update` was the trick which I was missing - will clean it up!